serverless/serverless
# Add Usage Plan to Amazon API Gateway

## What did you implement:

This PR adds usage plan to Amazon API Gateway.

Closes #2450
Closes #2741



## How did you implement it:
By adding `AWS::ApiGateway::UsagePlan` and `AWS::ApiGateway::UsagePlanKey` to Amazon API Gateway build flow.


## How can we verify it:

Create a service with API key(s) and verify that deployment creates usage plan and usage plan key(s) to CF stack and AWS.



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
